mujoco_warp/_src/collision_primitive.py

```diff
@@ -1718,6 +1730,497 @@ def capsule_box(
     )
 
 
+@wp.func
+def compute_rotmore(face_idx: int) -> wp.mat33:
```

Comment:
nit: let's prepend the name with an underscore as we probably don't expect others to use this outside this module, no?